src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/AccessTokenHandler.cs

```diff
@@ -0,0 +1,145 @@
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Common.AccessToken.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Altinn.Common.AccessToken
+{
+    public class AccessTokenHandler : AuthorizationHandler<AccessTokenRequirement>
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly ILogger _logger;
+        private readonly AccessTokenSettings _accessTokenSettings;
+        private readonly ISigningKeyResolver _signingKeysRetriever;
```

Comment:
Field is called retriever while class is called resolver.

Reply:
solved